app/src/androidTest/java/com/github/houseorganizer/houseorganizer/TaskListOnMainScreenTest.java

```diff
@@ -56,6 +56,7 @@ public static void createMockFirebase() throws ExecutionException, InterruptedEx
         FirebaseTestsHelper.setUpFirebase();
 
         auth = FirebaseAuth.getInstance();
+        FirestoreTaskTest.createMockFirebase();
```

Comment:
I don't know if this call is really necessary (line 59). How did it help when adding it?

Reply:
It helped because I kept having an error in this class that said that db was null. When adding the line it allowed db to be initialized thus not having the error anymore.
Maybe it was just a local thing but it fixed it on my machine ^^